aws/src/lint.js

```diff
@@ -84,28 +65,23 @@ function processResult(res, reject, resolve) {
   });
 }
 
-function buildHttpRequest(options, reject, resolve) {
-  return http.request(options, (res) => {
-    processResult(res, reject, resolve);
-  });
-}
-
-function buildHttpsRequest(options, reject, resolve) {
-  return https.request(options, (res) => {
+const sendHttpRequest = (options, postData) => new Promise((resolve, reject) => {
+  const req = http.request(options, (res) => {
     processResult(res, reject, resolve);
   });
-}
-
-const executeRequest = (collector, postData) => new Promise((resolve, reject) => {
-  const options = collector.lintEnv.createRequestOptions(collector.structure);
-  let req;
+  req.on('error', error => reject(error));
```

Comment:
Line#72 - lines#76 are duplicate of lines#85 - lines#93. Can we move these lines into a helper method? Say have processResult() as common method to process response from sendHttpsRequest(), sendHttpRequest() methods. Let sendHttpsRequest/sendHttpRequest make https/http calls and in turn call processResult().

Reply:
Sure. Will make the change.